scss/modules/_breadcrumbs.scss

```diff
@@ -0,0 +1,202 @@
+////
+/// @author       Web Team at Canonical Ltd
+/// @link         http://ubuntudesign.github.io/vanilla-framework/docs/#mixin-vf-lists
+/// @since        0.0.3
+////
+
+/// Default blockquote styling
+/// @group Blockquotes
+/// @example
+/// <blockquote>
+///   ...
+/// </blockquote>
+@mixin vf-breadcrumbs {
+  .nav-secondary {
+
+    @media only screen and (min-width: $breakpoint-large) {
+      border-bottom: 1px solid $alto-grey;
+    }
+
+    .breadcrumb {
+      box-sizing: border-box;
+      width: 100%;
+      margin-bottom: 0;
+      margin-left: 0;
+
+      @media only screen and (min-width: $breakpoint-large) {
+        width: auto;
+        float: left;
+        margin-left: 20px;
+      }
+
+      li {
+        display: block;
+        box-sizing: border-box;
+        width: 100%;
+        height: 40px;
+        margin: 0;
+        color: $white;
+
+        // scss-lint:disable NestingDepth
+        &:first-of-type {
+          border-bottom: 1px solid $alto-grey;
+
+          @media only screen and (min-width: $breakpoint-large) {
+            border-bottom: 0;
+          }
+        }
+
+        a {
+          font-size: 1em;
+          display: block;
+          box-sizing: border-box;
+          width: 100%;
+          margin-right: 0;
+          padding: 8px 10px 0;
```

Comment:
The links look closer to the top then bottom. I think if this is set to 10px 10px 0 it vertically centered.